.github/workflows/build-assets.yml

```diff
@@ -330,8 +330,7 @@ jobs:
           build-args: |
             PKG_FILE=${{ steps.ln-container.outputs.k8s_pkg_file }}
           tags: |
-            ${{ secrets.JFROG_REGISTRY || 'dummy.io' }}/ln/platform-core-${{ matrix.os }}:${{ needs.preamble.outputs.internal_tag }}
-            ${{ secrets.JFROG_REGISTRY || 'dummy.io' }}/ln/platform-core-${{ matrix.os }}:${{ needs.preamble.outputs.candidate_base_branch }}
+            ${{ secrets.JFROG_REGISTRY || 'dummy.io' }}/${{ secrets.REPOSITORY || 'hpccpl-docker-local' }}/hpcc-platform/${{ matrix.os }}/platform-core-ln:${{ needs.preamble.outputs.internal_tag }}
```

Comment:
I think the secrets.REPOSITORY is entirely unnecessary here after giving it another look. There's no way for the push to occur unless we're 'hpcc-systems' user. Lets just replace it with the hpccpl-docker-local path.